dev/Interactions/ButtonInteraction/ButtonInteraction.idl

```diff
@@ -17,7 +17,7 @@ runtimeclass ButtonInteractionInvokedEventArgs
 [WUXC_VERSION_PREVIEW]
 [webhosthidden]
 unsealed runtimeclass ButtonInteraction :
-#ifdef USE_INSIDER_SDK
+#ifdef USE_INTERNAL_SDK
```

Comment:
Please add build support for this new flag in the same way that UseInsiderSDK had it.

Reply:
It looks like the changes are in environment.props and in Build.cmd. Working on it.